.github/workflows/build_main_documentation.yml

```diff
@@ -1,5 +1,8 @@
 name: Build PR Documentation
 
+env:
+  DIFFUSERS_SLOW_IMPORT: yes
```

Comment:
double checking that `DIFFUSERS_SLOW_IMPORT=yes` means there will be no lazy loading. Correct?

Reply:
Yeah exactly

Reply:
https://github.com/huggingface/diffusers/pull/5176

Reply:
I can test then easily on this PR